htdocs/contact/consumption.php

```diff
@@ -398,14 +398,14 @@
 
     // Titles with sort buttons
     print '<tr class="liste_titre">';
-    print_liste_field_titre('Ref', $_SERVER['PHP_SELF'], 'doc_number', '', $param, 'class="left"', $sortfield, $sortorder);
```

Comment:
$param, **'',** $sortfield, $sortorder, 'left '

Reply:
you forgot to let , ''

Reply:
Done.